src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -0,0 +1,45 @@
+import { loc, createCallout } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'okta-verify-resend-push',
+
+    title () {
+      return loc('oie.okta_verify.push.title', 'login');
+    },
+
+    save () {
+      return loc('oie.okta_verify.push.send', 'login');
+    },
+
+    showMessages () {
+      // override showMessages to display error in cases like reject push
+      // or timeout. Borrowed this logic from TerminalView.
+      const messagesObjs = this.options.appState.get('messages');
+      if (messagesObjs && Array.isArray(messagesObjs.value)) {
+        this.add('<div class="ion-messages-container"></div>', '.o-form-error-container');
+
+        messagesObjs.value.forEach(messagesObj => {
+          const msg = messagesObj.message;
+          if (messagesObj?.class === 'ERROR') {
```

Comment:
`nit:` On a second thought, is it possible for `messagesObjs.value` to have falsy value like this `[{class: 'ERROR', message: 'error msg'}, undefined]` ? If it's empty (`[]`), it's taken care of by `forEach`. 
If the above condition holds good, this can indeed by simplified as -
```
const {message: msg, class} = messagesObj;

// class can be undefined if not present, but it is taken care of by ===
if (class === 'ERROR') {
.
.
```
This is just a `nit`, I'm good if it's not done as well.

Reply:
👍  Will handle this in the JIRA that takes care of messages on baseform